### ISSUE TITLE
[JS]: AJAX in JavaScript

### ISSUE THREAD

**Issue by aniketpathak028** [girlscript/winter-of-contributing#2358]
### Description

# Description
---
Creating a tutorial on AJAX in Javascript and discussing the following:

- What is AJAX?
- How does AJAX Work?
- AJAX - The XMLHttpRequest Object
- Fetch API

# Directory
---
``` Javascript/Topics/13.AJAX ```

### Domain

JavaScript

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by aniketpathak028** [comment]
@paramsiddharth pls assign me this issue 🙂

**Comment by Surbhi-sinha** [comment]
I wanted to add the section of AJAX XML and AJAX Applications. please assign it to me.
/assign

**Comment by paramsiddharth** [comment]
> I wanted to add the section of AJAX XML and AJAX Applications. please assign it to me. /assign

@Surbhi-sinha Please note that a PR for documentation has already been merged. Either extend it, or contribute in another format in your PR. Good luck!

**Comment by Surbhi-sinha** [comment]
> > I wanted to add the section of AJAX XML and AJAX Applications. please assign it to me. /assign
> 
> @Surbhi-sinha Please note that a PR for documentation has already been merged. Either extend it, or contribute in another format in your PR. Good luck!

I would definitely extend it. @paramsiddharth

**Comment by deepak089** [comment]
@paramsiddharth please this to me

**Comment by paramsiddharth** [comment]
> can you guide me how to submit my documentation @paramsiddharth

Of course, @deepak089 ! Please tag me in the General channel for JS in Discord. I'll help you out.
